File: backend/internal/constants/conflict.go
package constants

// ConflictStatus 利益冲突检查状态枚举。
const (
	ConflictStatusNoConflict    = "no_conflict"    // 未命中，可直接办理
	ConflictStatusPendingReview = "pending_review" // 命中，仅可保存为待复核
	ConflictStatusReleased      = "released"       // 管理员填写依据后放行（仍绑定档案版本）
	ConflictStatusRejected      = "rejected"       // 管理员驳回，不得办理
	ConflictStatusInvalidated   = "invalidated"    // 绑定档案变化，旧放行/结论失效
)

// ConflictStatusValues 全部冲突检查状态值。
var ConflictStatusValues = []string{
	ConflictStatusNoConflict, ConflictStatusPendingReview, ConflictStatusReleased,
	ConflictStatusRejected, ConflictStatusInvalidated,
}

// IsValidConflictStatus 校验冲突检查状态。
func IsValidConflictStatus(s string) bool {
	for _, v := range ConflictStatusValues {
		if v == s {
			return true
		}
	}
	return false
}

// 命中方式。
const (
	MatchedByIDNumber = "id_number"
	MatchedByName     = "name"
)

File: frontend/src/constants/conflict.ts
// 利益冲突检查状态/命中方式枚举（与后端 backend/internal/constants/conflict.go 保持一致）
export const ConflictStatus = {
  NO_CONFLICT: 'no_conflict',
  PENDING_REVIEW: 'pending_review',
  RELEASED: 'released',
  REJECTED: 'rejected',
  INVALIDATED: 'invalidated',
} as const

export const ConflictStatusText: Record<string, string> = {
  [ConflictStatus.NO_CONFLICT]: '无冲突·可办理',
  [ConflictStatus.PENDING_REVIEW]: '命中·待复核',
  [ConflictStatus.RELEASED]: '已放行',
  [ConflictStatus.REJECTED]: '已驳回',
  [ConflictStatus.INVALIDATED]: '已失效',
}

export const ConflictStatusColor: Record<string, string> = {
  [ConflictStatus.NO_CONFLICT]: 'green',
  [ConflictStatus.PENDING_REVIEW]: 'orange',
  [ConflictStatus.RELEASED]: 'blue',
  [ConflictStatus.REJECTED]: 'red',
  [ConflictStatus.INVALIDATED]: 'default',
}

export const ConflictStatusOptions = Object.entries(ConflictStatusText).map(([value, label]) => ({
  value,
  label,
}))

export const PartySideText: Record<string, string> = {
  our: '本方',
  opposing: '对方',
}

export const MatchedByText: Record<string, string> = {
  id_number: '证件号命中',
  name: '姓名命中',
}
